### PR TITLE
Feat/sexdropdown

### PR DESCRIPTION
**Pull request by matthew-wright**

### CODE REVIEW

File: src/app/edit-profile/edit-profile.page.html [hunk 1]
```diff
@@ -41,12 +44,22 @@
...
+              <ion-label position="fixed">
+                Sex
+              </ion-label>
+              <ion-select placeholder="Select One" name="sex" [(ngModel)]="sex" required>
+                <ion-select-option value="Female">Female</ion-select-option>
+                <ion-select-option value="Male">Male</ion-select-option>
+              </ion-select>
+            </ion-item>
+
+            <!-- <ion-item [hidden]="needVerification">
```
Comment (ryan-gontarek) [inline src/app/edit-profile/edit-profile.page.html]: Can delete?

File: src/app/sign-up/sign-up.page.html [hunk 2]
```diff
@@ -57,12 +57,22 @@
...
+                <ion-label position="fixed">
+                  Sex
+                </ion-label>
+                <ion-select placeholder="Select One" name="sex" [(ngModel)]="sex" required>
+                  <ion-select-option value="Female">Female</ion-select-option>
+                  <ion-select-option value="Male">Male</ion-select-option>
+                </ion-select>
+              </ion-item>
+
+            <!-- <ion-item [hidden]="needVerification">
```
Comment (ryan-gontarek) [inline src/app/sign-up/sign-up.page.html]: can delete the commented out section?

**Review by ryan-gontarek — COMMENTED** [review]
It looks good to me! Maybe delete the commented out sections and then we're good to go?

File: src/app/edit-profile/edit-profile.page.html (hunk 1 above)
Comment (matthew-wright) [reply src/app/edit-profile/edit-profile.page.html]: Yep, I noticed that too and I've deleted them. Thanks!

**Pull request merged**